torch/csrc/jit/codegen/cuda/lower_trivial_broadcast.cpp

```diff
@@ -14,8 +14,9 @@ void ConcretizedBroadcastDomains::build(Fusion* fusion) {
   exact_map_ = std::make_unique<ExactRootDomainMap>(fusion);
 
   // Initialize the origin map with input broadcast domains
+  auto inputs = fusion->inputsAndCreated();
```

Comment:
Are there other cases like this? Do we need to audit all `fusion->inputs()` calls?

Reply:
I skimmed through our codebase and feel that many of them are hard to tell if it needs to handle this or not. So I only changed the one that has a failing test.

For examples:

In the reduction and normalization scheduler, I see:
```C++
  auto tv_inps = ir_utils::filterByType<TensorView>(fusion->inputs());
  TORCH_INTERNAL_ASSERT(
      !tv_inps.empty(),
      "Tried to schedule a fusion with no tensor inputs, currently not supported.");
```
we should change this, but I don't know how much work is needed.